easy_style.gemspec

```diff
@@ -1,32 +1,32 @@
-require_relative 'lib/easy_style/version'
+require_relative "lib/easy_style/version"
 
 Gem::Specification.new do |spec|
-  spec.name          = 'easy_style'
+  spec.name          = "easy_style"
   spec.version       = EasyStyle::VERSION
-  spec.authors       = ['Easy Software Ltd']
-  spec.email         = ['info@easyredmine.com']
+  spec.authors       = ["Easy Software Ltd"]
+  spec.email         = ["info@easyredmine.com"]
 
-  spec.summary       = 'Rubocop configs'
-  spec.description   = 'Rubocop configs'
-  spec.homepage      = 'https://github.com/easysoftware/easy_style'
-  spec.license       = 'MIT'
+  spec.summary       = "Rubocop configs"
+  spec.description   = "Rubocop configs"
+  spec.homepage      = "https://github.com/easysoftware/easy_style"
+  spec.license       = "MIT"
   spec.required_ruby_version = ">= 3.1.2"
 
   # spec.metadata['allowed_push_host'] = 'TODO: Set to \'http://mygemserver.com\''
 
-  spec.metadata['homepage_uri'] = spec.homepage
-  spec.metadata['source_code_uri'] = spec.homepage
-  spec.metadata['changelog_uri'] = spec.homepage
+  spec.metadata["homepage_uri"] = spec.homepage
+  spec.metadata["source_code_uri"] = spec.homepage
+  spec.metadata["changelog_uri"] = spec.homepage
 
   # Specify which files should be added to the gem when it is released.
   # The `git ls-files -z` loads the files in the RubyGem that have been added into git.
   spec.files = Dir.chdir(File.expand_path(__dir__)) do
     `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
   end
-  spec.bindir        = 'exe'
+  spec.bindir        = "exe"
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
-  spec.require_paths = ['lib']
+  spec.require_paths = ["lib"]
 
-  spec.add_dependency 'rubocop', '~> 1.53.0'
-  spec.add_dependency 'rubocop-rails', '~> 2.20.0'
+  spec.add_dependency "rubocop", "~> 1.60.0"
```

Comment:
pridavaji novy copy, takze name neco zacne padat, takze bude potreba bud pak upravit default.yml pokud ty novy copy nebudou davat smysl. Nebo proste pregenerovat rubocop_todo ale to uz docela zvladame :)